workflow/rules/malt.smk

```diff
@@ -68,35 +68,56 @@ rule Malt_QuantifyAbundance:
     benchmark:
         "benchmarks/MALT_QUANTIFY_ABUNDANCE/{sample}.benchmark.txt"
     message:
-        "QUANTIFYING MICROBIAL ABUNDANCE USING MALT ALIGNMENTS FOR SAMPLE {input.sam}"
+        "QUANTIFYING MICROBIAL ABUNDANCE USING MALT SAM-ALIGNMENTS FOR SAMPLE {input.sam}"
     shell:
         """n_species=0; touch {output.counts}; for i in $(cat {params.unique_taxids}); do zgrep \"|tax|$i\" {input.sam} | grep -v '@' | awk '!a[$1]++' | wc -l >> {output.counts} || true; n_species=$((n_species+1)); echo Finished $n_species species; done &> {log}"""
 
 
-rule Malt_AbundanceMatrix:
+rule Malt_AbundanceMatrix_Sam:
```

Comment:
Reminder: if you change/add rule names, remember to update section `envmodules` in config.schema.yaml:

```
  envmodules:
    type: object
    description: >-
      envmodules listings for relevant rules
    default: {}
    properties:
      Malt_AbundanceMatrix_Sam:
        default: []
        $ref: "#/definitions/envmodules_list"
      Malt_AbundanceMatrix_Rma6:
        ...
```
and so on